### PR TITLE
Create CloudEvent data for three events

### PR DESCRIPTION
**Pull request by juliehockett**
Adds CE data for firebase-db1, legacy_pubsub, and pubsub_binary.

Also standardizes event-id, project-id, and timestamp across events.

### CODE REVIEW

**Comment by juliehockett** [comment]
Once these look good, I'll do the rest. Just want to thrash out any obvious issues before committing the time.

File: events/generate/data/firebase-db1-cloudevent-input.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (grant) [inline events/generate/data/firebase-db1-cloudevent-input.json:1]: In general this CloudEvent form is never produced by Eventarc, so it's not an accurate conformance test. Eventarc produces binary CloudEvents, so you'l see all of the CloudEvent metadata in HTTP headers.

Not sure how we generated this.

Even if a FF passes the conformance test for a structured CloudEvent, that doesn't guarantee actual conformance.

File: events/generate/data/firebase-db1-cloudevent-input.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "specversion": "1.0",
+  "type": "google.firebase.database.document.v1.written",
+  "source": "//firebase.googleapis.com/projects/_/instances/my-project-id/refs/gcf-test/xyz",
+  "id": "aaaaaa-1111-bbbb-2222-cccccccccccc",
+  "time": "2020-05-26T10:42:27.088Z",
+  "datacontenttype": "application/json",
+  "data": {
+    "data": null,
```
Comment (grant) [inline events/generate/data/firebase-db1-cloudevent-input.json:9]: Can we provide non-null data to verify this input works appropriately?

**Review by grant — APPROVED** [review]
These don't look like CloudEvents from Eventarc, added one comment, o.w. LGTM.

**Comment by jskeet** [comment]
Side note: this looks like it's one pull request that should be two; one for flags and one for the CloudEvents. (I'd also suggest using rebasing rather than merging in order to bring things in line with the default branch.) Will review just the third commit.

**Comment by juliehockett** [comment]
Ick sorry -- some thing clearly got confused in my checkout. The flags changes should've already been merged (though I should also have properly switched branches...). I overrwrote it all.

File: events/generate/data/firebase-db1-cloudevent-input.json (hunk 2 above)
Comment (juliehockett) [reply events/generate/data/firebase-db1-cloudevent-input.json:9]: I just used the existing firebase-db1 event -- if you have data that you would suggest, I can replace it.

File: events/generate/data/firebase-db1-cloudevent-input.json (hunk 1 above)
Comment (juliehockett) [reply events/generate/data/firebase-db1-cloudevent-input.json:1]: This is merely the test data representation -- how else would you propose to represent it? We create a CloudEvent from this, and send it via the Go SDK, which defaults to sending the binary encoding (https://github.com/cloudevents/sdk-go/blob/master/v2/binding/write.go#L86).

**Review by hdp617 — APPROVED** [review]

**Comment by juliehockett** [comment]
@hdp617 -- one thing that occurred to me is that this needs to handle the multiple versions of the pubsub/storage legacy events such that the validation for CloudEvent -- > legacy event always resolves to the correct one (and a decision will have to be made as to what that format is...Jon is probably the best resource to advise on that?)

**Comment by hdp617** [comment]
@juliehockett Just to make sure I understand this correctly, are you referring to the pubsub/storage legacy events in [0]?

For the CloudEvent -> legacy events conversion, we need to decide to convert a pubsub CloudEvent to a pubsub legacy event with
type `providers/cloud.pubsub/eventTypes/topic.publish`
or
type `google.pubsub.topic.publish`.

I think always converting to the latter one is fine since the doc says that "the legacy notifications might be removed at a future date." Unless some user's code relies on the type to be `providers/cloud.pubsub/eventTypes/topic.publish`.

@grant @jskeet What do you think? 

[0] https://cloud.google.com/functions/docs/calling/pubsub#legacy_cloud_pubsub_triggers

**Comment by grant** [comment]
I'd think we'd use `providers/cloud.pubsub/eventTypes/topic.publish`, but I haven't dived into details for this conversion.

**Comment by hdp617** [comment]
Any other thoughts before I merge?

File: events/generate/data/firebase-db1-cloudevent-input.json (hunk 1 above)
Comment (grant) [reply events/generate/data/firebase-db1-cloudevent-input.json:1]: This is fine and more ideal in terms of a format than a binary representation.

It's just a bit confusing how this is in the `events/generate` folder.

File: events/generate/data/firebase-db1-cloudevent-input.json (hunk 2 above)
Comment (grant) [reply events/generate/data/firebase-db1-cloudevent-input.json:9]: This works, nvm.

**Comment by grant** [comment]
LGTM from me.

**Pull request merged**